gensim/models/ldamodel.py

```diff
@@ -182,6 +187,366 @@ def load(cls, fname, *args, **kwargs):
         return result
 # endclass LdaState
 
+# DD
+# wrapper to just return lda.state after performing lda
+from distributed import local_client
+
+
+class DaskWorker(object):
```

Comment:
it's better to place classes in distinct place (like with `Pyro4`)

Reply:
I agree. This was done to speed up development, as the code needs to copied to the dask workers file system, as the python file itself isn't pickled and sent.